src/Facets/GasZipFacet.sol

```diff
@@ -107,7 +110,7 @@ contract GasZipFacet is ILiFi, ReentrancyGuard, SwapperV2, Validatable {
         if (
             _bridgeData.receiver != NON_EVM_ADDRESS &&
             _gasZipData.receiverAddress !=
-            bytes32(uint256(uint160(_bridgeData.receiver)))
+            bytes32(bytes20(uint160(_bridgeData.receiver)))
```

Comment:
wondering if we should add a comment here to make it clear that we need to use this way of conversion cause GasZip requires right-padded bytes32 value. 
WDYT ?